sci-geosciences/grass/grass-8.2.0-r1.ebuild

```diff
@@ -0,0 +1,286 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..10} )
+PYTHON_REQ_USE="sqlite"  # bug 572440
+WX_GTK_VER="3.0-gtk3"
+
+inherit autotools desktop python-single-r1 toolchain-funcs wxwidgets xdg
+
+DESCRIPTION="A free GIS with raster and vector functionality, as well as 3D vizualization"
+HOMEPAGE="https://grass.osgeo.org/"
+
+LICENSE="GPL-2"
+
+GVERSION=${SLOT#*/}
+MY_PM="${PN}${GVERSION}"
+MY_PM="${MY_PM/.}"
+
+if [[ ${PV} =~ "9999" ]]; then
+	inherit git-r3
+	SLOT="0/8.3"
```

Comment:
Shouldn't this be 8.2 instead of 8.3?

Reply:
This is intentional. Odd minor version are used by upstream for development branches. So after releasing 8.2.0 the SLOT for development versions has to be changed to 8.3. But this only matters for the live ebuild. These changes were made just as a matter of form, so it would reflect the state of the live ebuild at the release time of the corresponding version.